sunxspex/emission.py

```diff
@@ -284,8 +291,8 @@ def bremsstrahlung_cross_section(electron_energy, photon_energy, z=1.2):
     return cross_section
 
 
-def get_integrand(*, model, electron_energy, photon_energy, eelow, eebrk, eehigh, p, q, z=1.2,
-                  efd=True):
+def _get_integrand0(*, model, electron_energy, photon_energy, eelow, eebrk, eehigh, p, q, z=1.2,
```

Comment:
Is this used anywhere or has it just been renamed to keep a copy of the old `get_integrand`?